examples/distributed_autofaiss_n_indices.py

```diff
@@ -56,3 +54,8 @@
 sorted_k_ids = np.take_along_axis(knn_ids_arr, sorted_k_indices, axis=1)
 print(f"{K} nearest distances: {sorted_k_dists}")
 print(f"{K} nearest ids: {sorted_k_ids}")
+
+# also consider using faiss.merge_ondisk (https://github.com/facebookresearch/faiss/blob/30abcd6a865afef7cf86df7e8b839a41b5161505/contrib/ondisk.py )
+# https://github.com/facebookresearch/faiss/blob/151e3d7be54aec844b6328dc3e7dd0b83fcfa5bc/demos/demo_ondisk_ivf.py
+# to merge indices on disk without using memory, and then use a very large index with almost no memory usage.
+# required if the number of indices piece makes the overhead (around 500MB) of loading memmaped indices too large.
```

Comment:
I think that it would be more pratical to write directly the example for the 3rd use case here (with the original references in comment).
The example  in your PR description is quite good already, just need to adapt a bit the code maybe.

It can begin with:
\###############################
\# Use case 3: merging N indices on disk #
\###############################

Reply:
good point, I changed it